src/components/ui/input.tsx

```
@@ -2,22 +2,64 @@
     const [showPassword, setShowPassword] = React.useState(false);
+    const [hasFiles, setHasFiles] = React.useState(false);
 
     // Variables
     const isPassword = type === "password";
+    const isFile = type === "file";
+    const id = React.useId();
+
+    // Safely get error from form field if available
+    let error: FieldError | undefined;
+    try {
+        const fieldContext = useFormField();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)